src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const addOption = 'addProperties';
+  const removeOption = 'removeProperties';
+  const clearOption = 'clear';
```

Comment:
it's better to use uppercase naming for constants
```suggestion
  const ADD_OPTION = 'addProperties';
  //...
```